user/private-dependencies.md

```diff
@@ -199,6 +200,22 @@ gem 'lib1', github: "myorg/lib1"
 gem 'lib2', github: "myorg/lib2"
 ```
 
+
+**Note**: In case of private git submodules, be aware that the `git submodule
```

Comment:
And then see if you can make the code block a child of the note? 8 spaces might do the trick :-p